client/my-sites/earn/memberships/style.scss

```diff
@@ -307,93 +307,6 @@
 	float: right;
 }
 
-.memberships__dialog-nav.section-nav {
-	box-shadow: none;
-	border-bottom: 1px solid var(--color-neutral-5);
-}
-
-.memberships__dialog-sections {
-	display: grid;
-	grid-template: "item" 1fr / 1fr;
-
-	* {
-		font-size: $font-body-small;
-	}
-
-	@include break-medium {
-		min-width: 580px;
-	}
-	.memberships__dialog-advanced-options {
-		width: 100%;
-		box-shadow: none;
-		.foldable-card__header {
-			padding: 0;
-		}
-		&.foldable-card.is-expanded {
-			margin: 0;
-			.foldable-card__content {
-				border-top: none;
-				padding: 0;
-			}
-		}
-		.foldable-card__main {
-			display: block;
-			font-weight: 600;
-			.foldable-card__action {
-				right: unset;
-			}
-		}
-		fieldset {
-			margin-bottom: 0;
-		}
-
-		.foldable-card__expand .gridicon {
-			fill: var(--color-text);
-		}
-	}
-
-	.notice__content {
-		max-width: 500px;
-	}
-}
-
```

Comment:
The css below this comment look like it was originally added for this same plans dialog. I searched the rest of the earn page, and these css selectors no longer exist anywhere in the markup. So I'm assuming this css (which is years old) is just no longer needed. If anyone sees otherwise, let me know. 

I'm hesitant to remove css, but if I'm refactoring anyways, I also don't want to just leave 'dead' css here in the file. 

This applies to:
memberships__dialog-section
memberships__dialog-form-header
memberships__dialog-sections-price